client-spark/spark3/src/main/java/org/apache/spark/shuffle/reader/RssShuffleReader.java

```diff
@@ -210,7 +218,8 @@ public Configuration getHadoopConf() {
         CreateShuffleReadClientRequest request = new CreateShuffleReadClientRequest(
             appId, shuffleId, partition, storageType, basePath, indexReadLimit, readBufferSize,
             1, partitionNum, partitionToExpectBlocks.get(partition), taskIdBitmap, shuffleServerInfoList,
-            hadoopConf, dataDistributionType, expectedTaskIdsBitmapFilterEnable);
+            hadoopConf, dataDistributionType, expectedTaskIdsBitmapFilterEnable,
```

Comment:
If we have `blockSkipStrategy`, could we remove `expectedTaskIdsBitmapFilterEnable`?

Reply:
No, if we don't use AQE and we use one replica, it is also need.

Reply:
What's situation that we use `expectedTaskIdsBitmapFilterEnable`? If we don't use aqe and multi replicas, the taskbitmap will occupy several KB memory. But we could filter less data.

Reply:
You are right. But not taskbitmap don't support multi replicas now. Should we support it in this pr?

Reply:
No. We can implement it in another pr. We can forbid taskbitmap filter in multi replicas. Just throw a `NotSupportException`.

Reply:
I thought again. if client don't use AQE and use one replica, we can set `BlockSkipStrategy` to `NONE` automatically.

Reply:
> I thought again. if client don't use AQE and use one replica, we can set `BlockSkipStrategy` to `NONE` automatically.

It's ok for me, too.

Reply:
So should we remove `expectedTaskIdsBitmapFilterEnable`?

Reply:
> So should we remove `expectedTaskIdsBitmapFilterEnable`?

Yes. Done.